packages/chat/components/chat/__stories__/chat.mdx

```diff
@@ -0,0 +1,138 @@
+import { ArgTypes, Markdown, Meta } from '@storybook/blocks';
+import { cdnJs, cdnCss } from '../../../../../globals/internal/storybook-cdn';
+import * as ChatStories from './chat.stories';
+import packageJson from '../../../package.json';
+
+<Meta of={ChatStories} />
+
+# Chat
+
+> 💡 Check our
+> [Stackblitz](https://stackblitz.com/github/carbon-design-system/carbon-labs/tree/main/packages/chat/examples/chat)
+> example implementation.
```

Comment:
we'll need to add in an `examples` folder in the repo in order for this stackblitz link to work. I'll go ahead and approve since I don't want this to be blocking any work. The `examples` folder can be added in the next PR